recipes/pyspark/pysparkretailapp/helper.py

```diff
@@ -22,6 +22,7 @@
 from pyspark.sql.functions import unix_timestamp, from_unixtime, to_date, lit, lag, udf, date_format
 from pyspark.sql import Window
 import datetime
+import com.adobe.platform.query.QSOptimizations
```

Comment:
This isn't going to work.  You have to do this the same like we did with the Dataset Options (I'm pretty sure)
```
def get_optimizations():
  return spark_context._jvm.com.adobe.platform.query.QSOptimizations
```

Reply:
Thinking about this more, we don't want to do this here anyway.

Reply:
yes i copied it from scala and forgot to remove it here